projects/packages/forms/tests/js/contact-form/contact-form.test.js

```diff
@@ -126,4 +127,36 @@ describe( 'Contact Form', () => {
 			expect( spy ).not.toHaveBeenCalled();
 		} );
 	} );
+	// @see https://github.com/Automattic/jetpack/issues/41834.
+	it( 'should properly handle `select` elements when checking if a form is empty', async () => {
+		setFormContent( `
+			<div style="" class="grunion-field-select-wrap grunion-field-wrap">
```

Comment:
I'm not an expert in testing components, but it seems weird to me that we add arbitrary html in order to test `accessible-form.js`. Should those tests be `e2e` tests?